source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +235,64 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', () => {
+      const leaveAppRunning = Boolean(global.config.get('system.leaveAppRunning'))
+      if (leaveAppRunning) {
+        if (process.platform === 'darwin') {
+          if (this._tray == null) {
+            let basepath = app.getAppPath()
+            this._tray = new Tray(path.join(__dirname, './assets/icons/128x128.png'))
+
+            const contextMenu = Menu.buildFromTemplate([
+              {
+                label: 'Show Zettlr',
+                click: () => {
+                  this.showAnyWindow()
+                },
+                type: 'normal'
+              },
+              { label: '', type: 'separator' },
+              {
+                label: 'Quit',
+                click: () => {
+                  app.quit()
+                },
+                type: 'normal'
+              }
+            ])
+            this._tray.setToolTip('This is the Zettlr tray. \n Select Show Zettlr to show the Zettlr app. \n Select Quit to quit the Zettlr app.')
+            this._tray.setContextMenu(contextMenu)
+          }
+        } else {
+          if (this._tray == null) {
+            this._tray = new Tray(path.join(__dirname, 'assets/icons/128x128.png'))
```

Comment:
Windows should use `icon.ico` as per the [tray api](https://www.electronjs.org/docs/api/tray).

On Windows (only) this line should be:
```typescript
            this._tray = new Tray(path.join(__dirname, 'assets/icons/icon.ico'))
```

For Linux use the code you have:
```typescript
            this._tray = new Tray(path.join(__dirname, 'assets/icons/128x128.png'))
```
Later #17 can install and setup the linux `128x128.png` file.

Reply:
will create icon.ico

Reply:
add function only windows show icon.ico